tests/test_buck_integration.py

```diff
@@ -63,27 +63,27 @@ def verify_genproj(self, lib_name, mod_count, app_path):
     def test_flat_genproj(self):
         app_path = join(tempfile.gettempdir(), 'apps', 'mockapp')
         args = [
-            "--output_directory", app_path, "--buck_module_path", "/apps/mockapp", "--gen_type", "flat",
+            "--output_directory", app_path, "--blaze_module_path", "/apps/mockapp", "--gen_type", "flat",
             "--lines_of_code", "150000"
         ]
         command = GenProjCommandLine()
         command.main(args)
 
-        self.verify_genproj('MockLib53', 101, app_path)
+        self.verify_genproj('MockLib53', 102, app_path)
```

Comment:
we are now adding `.buckconfig` into the generated project, hence why this happens.